client/blocks/product-purchase-features-list/test/happiness-support-card.jsx

```diff
@@ -0,0 +1,35 @@
+jest.mock( 'components/happiness-support', () => 'HappinessSupport' );
```

Comment:
yes, makes sense to me. Thanks for pointing it out Jon. I'll remove the file at this stage, because the child component has its own tests, and the inclusion of the component in `ProductPurchaseList` is also tested elsewhere.